layouts/partials/fragments/portfolio.html

```diff
@@ -7,36 +7,6 @@
     {{- printf " bg-%s" $bg -}}
   ">
     <div class="container py-5">
-      {{- with .Params.title }}
```

Comment:
The specific title and subtitle code should still stay within each fragment. It should just be considered a new fragment, which consists of just title and subtitle.

Enabling something like:

features (header)
-
3 features (items)
-
1 specific feature (item)

So instead of linking to one of the specific feature sections one can link to the header.